biothings/tests/biothings_pytest_plugin/README.md

```diff
@@ -0,0 +1,52 @@
+# Biothings Pytest Plugin
+
+The `biothings_pytest_plugin` is a custom Pytest plugin designed to integrate Slack notifications and AWS S3 interactions into your testing workflow. It allows you to automate the process of sending test results to a Slack channel and managing application metadata stored in AWS S3.
```

Comment:
I would probably change the name to be slight more descriptive. If we add more plugins in the future we'll likely need more specific names. Something like biothings-slack-notifier potentially